test/common.js

```diff
@@ -423,8 +423,10 @@ function runCallChecks(exitCode) {
 }
 
 
-exports.mustCall = function(fn, expected) {
-  if (typeof expected !== 'number') expected = 1;
+exports.mustCall = function(fn, expected = 1) {
```

Comment:
Oh, I didn't realize this would be backported. Which is the preferred method - older syntax everywhere, or change during the backport?